diff --git a/kde/cursors/build_hyprcursors.sh b/kde/cursors/build_hyprcursors.sh
new file mode 100755
index 0000000..5c7d24b
--- /dev/null
+++ b/kde/cursors/build_hyprcursors.sh
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+generate_cursor() {
+	local rawsvg="$1"
+	local cursor="$2"
+	local dir="$3"
+
+	if [ "$dir/$cursor.svg" -ot $rawsvg ] ; then
+		inkscape -i $cursor $rawsvg --export-filename="$dir/$cursor.svg" > /dev/null
+	fi
+}
+
+generate_animated_cursor() {
+	local rawsvg="$1"
+	local cursor="$2"
+	local dir="$3"
+
+	rm "$dir/$cursor/$cursor.svg"
+	for i in 01 02 03 04 05 06 07 08 09 10 11 12 13 14 15 16 17 18 19 20 21 22 23
+	do
+		generate_cursor $rawsvg $cursor-$i $dir/$cursor
+	done
+}
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+RAWSVG="src/cursors.svg"
+INDEX="src/index.theme"
+ALIASES="src/cursorList"
+
+
+echo -ne "Checking Requirements...\\r"
+if [ ! -f $RAWSVG ] ; then
+	echo -e "\\nFAIL: '$RAWSVG' missing in /src"
+	exit 1
+fi
+
+if [ ! -f $INDEX ] ; then
+	echo -e "\\nFAIL: '$INDEX' missing in /src"
+	exit 1
+fi
+
+if  ! type "inkscape" > /dev/null ; then
+	echo -e "\\nFAIL: inkscape must be installed"
+	exit 1
+fi
+
+echo -e "Checking Requirements... DONE"
+
+DIR="build_hyprcursors/hyprcursors"
+
+echo -e "\033[0KGenerating simple cursor pixmaps..."
+for CUR in src/config/*.cursor; do
+	BASENAME=$CUR
+	BASENAME=${BASENAME##*/}
+	BASENAME=${BASENAME%.*}
+
+	generate_cursor $RAWSVG $BASENAME $DIR/$BASENAME
+done
+echo -e "\033[0KGenerating simple cursor pixmaps... DONE"
+
+echo -ne "\033[0KGenerating animated cursor pixmaps..."
+generate_animated_cursor $RAWSVG progress $DIR
+generate_animated_cursor $RAWSVG wait $DIR
+echo -e "\033[0KGenerating animated cursor pixmaps... DONE"
+
+echo -ne "\033[0KGenerating hyprcursors..."
+cd build_hyprcursors/
+hyprcursor-util -c .
+echo -ne "\033[0KGenerating hyprcursors... DONE"
+
+echo "COMPLETE!"
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/alias/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/alias/meta.hl
new file mode 100644
index 0000000..f976a4a
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/alias/meta.hl
@@ -0,0 +1,10 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, alias.svg, 50
+
+define_override = a2a266d0498c3104214a47bd64ab0fc8
+define_override = 3085a0e285430894940527032f8b26df
+define_override = 640fb0e74195791501fd1ed57b41487f
+define_override = link
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/all-scroll/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/all-scroll/meta.hl
new file mode 100644
index 0000000..58b530d
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/all-scroll/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, all-scroll.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/bottom_left_corner/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/bottom_left_corner/meta.hl
new file mode 100644
index 0000000..a2b8ba9
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/bottom_left_corner/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 1.08
+
+define_size = 64, bottom_left_corner.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/bottom_right_corner/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/bottom_right_corner/meta.hl
new file mode 100644
index 0000000..18b2c56
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/bottom_right_corner/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 1.08
+hotspot_y = 1.08
+
+define_size = 64, bottom_right_corner.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/bottom_side/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/bottom_side/meta.hl
new file mode 100644
index 0000000..cae5a0c
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/bottom_side/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 1.17
+
+define_size = 64, bottom_side.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/cell/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/cell/meta.hl
new file mode 100644
index 0000000..4580ef7
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/cell/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, cell.svg, 50
+
+define_override = plus
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/center_ptr/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/center_ptr/meta.hl
new file mode 100644
index 0000000..6f4d521
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/center_ptr/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.83
+hotspot_y = 0.17
+
+define_size = 64, center_ptr.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/col-resize/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/col-resize/meta.hl
new file mode 100644
index 0000000..b79a83e
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/col-resize/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, col-resize.svg, 50
+
+define_override = split_h
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/color-picker/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/color-picker/meta.hl
new file mode 100644
index 0000000..3eb2031
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/color-picker/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 1.21
+
+define_size = 64, color-picker.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/context-menu/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/context-menu/meta.hl
new file mode 100644
index 0000000..1884235
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/context-menu/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, context-menu.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/copy/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/copy/meta.hl
new file mode 100644
index 0000000..0ba2a85
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/copy/meta.hl
@@ -0,0 +1,10 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, copy.svg, 50
+
+define_override = 6407b0e94181790501fd1e167b474872
+define_override = b66166c04f8c3109214a4fbd64a50fc8
+define_override = 1081e37283d90000800003c07f3ef6bf
+define_override = dnd-copy
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/crosshair/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/crosshair/meta.hl
new file mode 100644
index 0000000..617b90c
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/crosshair/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, crosshair.svg, 50
+
+define_override = cross
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/default/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/default/meta.hl
new file mode 100644
index 0000000..ebe9917
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/default/meta.hl
@@ -0,0 +1,12 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, default.svg, 50
+
+define_override = left_ptr
+define_override = top_left_arrow
+define_override = size-fdiag
+define_override = size-bdiag
+define_override = size-ver
+define_override = size-hor
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/dnd-move/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/dnd-move/meta.hl
new file mode 100644
index 0000000..e2ac1bd
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/dnd-move/meta.hl
@@ -0,0 +1,13 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, dnd-move.svg, 50
+
+define_override = fcf21c00b30f7e3f83fe0dfd12e71cff
+define_override = 4498f0e0c1937ffe01fd06f973665830
+define_override = 9081237383d90e509aa00f00170e968f
+define_override = dnd-none
+define_override = grabbing
+define_override = move
+define_override = closedhand
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/dnd-no-drop/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/dnd-no-drop/meta.hl
new file mode 100644
index 0000000..b3c0661
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/dnd-no-drop/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, dnd-no-drop.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/down-arrow/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/down-arrow/meta.hl
new file mode 100644
index 0000000..2078057
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/down-arrow/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 1.17
+
+define_size = 64, down-arrow.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/draft/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/draft/meta.hl
new file mode 100644
index 0000000..88cb45e
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/draft/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 1.21
+
+define_size = 64, draft.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/fleur/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/fleur/meta.hl
new file mode 100644
index 0000000..b90bf9f
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/fleur/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, fleur.svg, 50
+
+define_override = size_all
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/help/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/help/meta.hl
new file mode 100644
index 0000000..8be9a69
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/help/meta.hl
@@ -0,0 +1,11 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, help.svg, 50
+
+define_override = question_arrow
+define_override = d9ce0ab605698f320427677b458ad60b
+define_override = 5c6cd98b3f3ebcb1f9c7f1c204630408
+define_override = whats_this
+define_override = left_ptr_help
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/left-arrow/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/left-arrow/meta.hl
new file mode 100644
index 0000000..dafa411
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/left-arrow/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.67
+
+define_size = 64, left-arrow.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/left_side/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/left_side/meta.hl
new file mode 100644
index 0000000..d7d5653
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/left_side/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.67
+
+define_size = 64, left_side.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/no-drop/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/no-drop/meta.hl
new file mode 100644
index 0000000..08718f6
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/no-drop/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, no-drop.svg, 50
+
+define_override = forbidden
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/not-allowed/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/not-allowed/meta.hl
new file mode 100644
index 0000000..791d866
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/not-allowed/meta.hl
@@ -0,0 +1,9 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, not-allowed.svg, 50
+
+define_override = circle
+define_override = 03b6e0fcb3499374a867c041f52298f0
+define_override = crossed_circle
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/openhand/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/openhand/meta.hl
new file mode 100644
index 0000000..3f3dc83
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/openhand/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, openhand.svg, 50
+
+define_override = grab
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/pencil/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/pencil/meta.hl
new file mode 100644
index 0000000..3366105
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/pencil/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 1.21
+
+define_size = 64, pencil.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/pirate/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/pirate/meta.hl
new file mode 100644
index 0000000..f9be24d
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/pirate/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, pirate.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/pointer/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/pointer/meta.hl
new file mode 100644
index 0000000..63e5d1f
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/pointer/meta.hl
@@ -0,0 +1,11 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.17
+
+define_size = 64, pointer.svg, 50
+
+define_override = hand2
+define_override = hand1
+define_override = pointing_hand
+define_override = e29285e634086352946a0e7090d73106
+define_override = 9d800788f1b08800ae810202380a0822
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/progress/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/progress/meta.hl
new file mode 100644
index 0000000..a7f8d9c
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/progress/meta.hl
@@ -0,0 +1,33 @@
+resize_algorithm = bilinear
+hotspot_x = 0.17
+hotspot_y = 0.17
+
+define_size = 64, progress-01.svg, 30
+define_size = 64, progress-02.svg, 30
+define_size = 64, progress-03.svg, 30
+define_size = 64, progress-04.svg, 30
+define_size = 64, progress-05.svg, 30
+define_size = 64, progress-06.svg, 30
+define_size = 64, progress-07.svg, 30
+define_size = 64, progress-08.svg, 30
+define_size = 64, progress-09.svg, 30
+define_size = 64, progress-10.svg, 30
+define_size = 64, progress-11.svg, 30
+define_size = 64, progress-12.svg, 30
+define_size = 64, progress-13.svg, 30
+define_size = 64, progress-14.svg, 30
+define_size = 64, progress-15.svg, 30
+define_size = 64, progress-16.svg, 30
+define_size = 64, progress-17.svg, 30
+define_size = 64, progress-18.svg, 30
+define_size = 64, progress-19.svg, 30
+define_size = 64, progress-20.svg, 30
+define_size = 64, progress-21.svg, 30
+define_size = 64, progress-22.svg, 30
+define_size = 64, progress-23.svg, 30
+
+define_override = half-busy
+define_override = 00000000000000020006000e7e9ffc3f
+define_override = 08e8e1c95fe2fc01f976f1e063a64ccd
+define_override = 3ecb610c1bf6410f44200f48c40d3599
+define_override = left_ptr_watch
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/right-arrow/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/right-arrow/meta.hl
new file mode 100644
index 0000000..e8cb135
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/right-arrow/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 1.17
+hotspot_y = 0.67
+
+define_size = 64, right-arrow.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/right_ptr/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/right_ptr/meta.hl
new file mode 100644
index 0000000..f8e82fe
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/right_ptr/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 1.17
+hotspot_y = 0.17
+
+define_size = 64, right_ptr.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/right_side/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/right_side/meta.hl
new file mode 100644
index 0000000..45b1a4b
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/right_side/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 1.17
+hotspot_y = 0.67
+
+define_size = 64, right_side.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/row-resize/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/row-resize/meta.hl
new file mode 100644
index 0000000..bc47e31
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/row-resize/meta.hl
@@ -0,0 +1,7 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, row-resize.svg, 50
+
+define_override = split_v
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/size_bdiag/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/size_bdiag/meta.hl
new file mode 100644
index 0000000..b1f0323
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/size_bdiag/meta.hl
@@ -0,0 +1,9 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, size_bdiag.svg, 50
+
+define_override = sw-resize
+define_override = nesw-resize
+define_override = ne-resize
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/size_fdiag/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/size_fdiag/meta.hl
new file mode 100644
index 0000000..535881d
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/size_fdiag/meta.hl
@@ -0,0 +1,9 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, size_fdiag.svg, 50
+
+define_override = se-resize
+define_override = nw-resize
+define_override = nwse-resize
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/size_hor/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/size_hor/meta.hl
new file mode 100644
index 0000000..968db05
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/size_hor/meta.hl
@@ -0,0 +1,11 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, size_hor.svg, 50
+
+define_override = sb_h_double_arrow
+define_override = w-resize
+define_override = e-resize
+define_override = h_double_arrow
+define_override = ew-resize
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/size_ver/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/size_ver/meta.hl
new file mode 100644
index 0000000..54e6ae7
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/size_ver/meta.hl
@@ -0,0 +1,12 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, size_ver.svg, 50
+
+define_override = s-resize
+define_override = ns-resize
+define_override = 00008160000006810000408080010102
+define_override = n-resize
+define_override = sb_v_double_arrow
+define_override = v_double_arrow
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/text/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/text/meta.hl
new file mode 100644
index 0000000..d9e400a
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/text/meta.hl
@@ -0,0 +1,8 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, text.svg, 50
+
+define_override = ibeam
+define_override = xterm
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/top_left_corner/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/top_left_corner/meta.hl
new file mode 100644
index 0000000..d0c861c
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/top_left_corner/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.21
+hotspot_y = 0.17
+
+define_size = 64, top_left_corner.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/top_right_corner/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/top_right_corner/meta.hl
new file mode 100644
index 0000000..f72d12c
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/top_right_corner/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 1.08
+hotspot_y = 0.17
+
+define_size = 64, top_right_corner.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/top_side/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/top_side/meta.hl
new file mode 100644
index 0000000..2feab0e
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/top_side/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.17
+
+define_size = 64, top_side.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/up-arrow/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/up-arrow/meta.hl
new file mode 100644
index 0000000..fe299c0
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/up-arrow/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.17
+
+define_size = 64, up-arrow.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/vertical-text/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/vertical-text/meta.hl
new file mode 100644
index 0000000..7aa1a5a
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/vertical-text/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, vertical-text.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/wait/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/wait/meta.hl
new file mode 100644
index 0000000..8da39aa
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/wait/meta.hl
@@ -0,0 +1,29 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, wait-01.svg, 30
+define_size = 64, wait-02.svg, 30
+define_size = 64, wait-03.svg, 30
+define_size = 64, wait-04.svg, 30
+define_size = 64, wait-05.svg, 30
+define_size = 64, wait-06.svg, 30
+define_size = 64, wait-07.svg, 30
+define_size = 64, wait-08.svg, 30
+define_size = 64, wait-09.svg, 30
+define_size = 64, wait-10.svg, 30
+define_size = 64, wait-11.svg, 30
+define_size = 64, wait-12.svg, 30
+define_size = 64, wait-13.svg, 30
+define_size = 64, wait-14.svg, 30
+define_size = 64, wait-15.svg, 30
+define_size = 64, wait-16.svg, 30
+define_size = 64, wait-17.svg, 30
+define_size = 64, wait-18.svg, 30
+define_size = 64, wait-19.svg, 30
+define_size = 64, wait-20.svg, 30
+define_size = 64, wait-21.svg, 30
+define_size = 64, wait-22.svg, 30
+define_size = 64, wait-23.svg, 30
+
+define_override = watch
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/wayland-cursor/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/wayland-cursor/meta.hl
new file mode 100644
index 0000000..4c53ea4
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/wayland-cursor/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, wayland-cursor.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/x-cursor/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/x-cursor/meta.hl
new file mode 100644
index 0000000..b40e9d8
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/x-cursor/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, x-cursor.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/zoom-in/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/zoom-in/meta.hl
new file mode 100644
index 0000000..1b20069
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/zoom-in/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, zoom-in.svg, 50
diff --git a/kde/cursors/build_hyprcursors/hyprcursors/zoom-out/meta.hl b/kde/cursors/build_hyprcursors/hyprcursors/zoom-out/meta.hl
new file mode 100644
index 0000000..5c23936
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/hyprcursors/zoom-out/meta.hl
@@ -0,0 +1,5 @@
+resize_algorithm = bilinear
+hotspot_x = 0.67
+hotspot_y = 0.67
+
+define_size = 64, zoom-out.svg, 50
diff --git a/kde/cursors/build_hyprcursors/manifest.toml b/kde/cursors/build_hyprcursors/manifest.toml
new file mode 100644
index 0000000..dc19771
--- /dev/null
+++ b/kde/cursors/build_hyprcursors/manifest.toml
@@ -0,0 +1,5 @@
+[General]
+name = "Sweet-cursors-hyprcursor"
+description = "Sweet cursor theme for hyprcursor"
+version = "0.1"
+cursors_directory = "hyprcursors"
